### DISCUSSION TITLE
Windows Setup (optional alternative) - PLEX Credentials

### DISCUSSION THREAD

**Discussion by CBarbas** [Taxel/PlexTraktSync#2004] [Support]
Hi all,

I am trying to setup PlexTraktSync via the Windows Setup but when I reach the step to enter my PLEX Password, after I have entered the PLEX Username or PLEX User Email, with both it is happening the same, I am not able to type my PLEX Password at the "setup.bat".

Thanks in advance

**Comment by glensc** [comment]
What do you mean not able to type password? Describe what is happening. 

Password is not printed on screen when you type it. Just press enter after you've input password.

**Comment by CBarbas** [comment]
Hi, at this step,

"Please enter your Plex username or e-mail (): xxxxxxx
Please enter your Plex password:"

I am not able to type the password and if we say I am typing but its not printed as you are saying, I am getting this after "enter",

```
Please enter your Plex username or e-mail (): xxxxxxx
Please enter your Plex password:
Enter a 2FA code if enabled, or leave blank otherwise:
Log in to Plex failed: '(401) unauthorized; https://plex.tv/api/v2/users/signin <?xml version="1.0"
encoding="UTF-8"?> <errors>   <error code="1001" message="User could not be authenticated" status="401"/> </errors> ',
Try again.
Please enter your Plex username or e-mail (xxxxxx):
```

I have replaced my username with "xxxxx" just here at the post.

**Reply by glensc** [reply]
You could try using some other terminal or other system to log in and then copy .env and config.yml and servers.yml

In case you're actually pasting password, try really typing it (blindly), the errors really look like you typed wrong password. does it contain characters outside ASCII? try changing password then. it could also be temporary error, but unlikely because seems you tried it multiple times over time.

**Comment by CBarbas** [comment]
No, pass is only numbers and letters, no matter if I type the pass or if I  CTRL+V the pass, it is happening the same.
And I forgot to mention that I have no 2FA enabled, so I am leaving it blank.

I'll try whats you are saying, I'll check if I am able to!

**Reply by glensc** [reply]
please reply in thread, not start a new thread.
